src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CFTTaskMapperTest.java

```diff
@@ -1145,6 +1148,9 @@ void should_map_task_resource_to_task_with_permissions_empty() {
         assertTrue(task.getPermissions().getValues().isEmpty());
         assertNotNull(task.getReconfigureRequestTime());
         assertNotNull(task.getLastReconfigurationTime());
+        assertNotNull(task.getLastUpdatedTimestamp());
```

Comment:
These should check the actual values not just if they're not null.

Reply:
done